### ISSUE TITLE
New acronyms should be spelled out the first time (and potentially linked to wikipedia, etc)

### ISSUE THREAD

**Issue by bakercp** [openframeworks/ofBook#160]
New users may not know about more advanced concepts / acronyms like "VBO" http://openframeworks.cc/ofBook/chapters/advanced_graphics.html.

**Comment by tpltnt** [comment]
@bakercp you are right and I am doing it during my reviews. This is already been done for the first 6 chapters I suppose.

**Comment by bakercp** [comment]
+1
